projects/core/src/routing/routing.module.ts

```diff
@@ -20,6 +21,7 @@ import { ROUTING_FEATURE } from './store/state';
     StoreModule.forFeature(ROUTING_FEATURE, reducerToken),
     EffectsModule.forFeature(effects),
     StoreRouterConnectingModule.forRoot({
+      routerState: RouterState.Minimal,
```

Comment:
Do we need this line if we use our custom serializer?

Reply:
Without it, this action fails:
<img width="446" alt="Screen Shot 2019-06-19 at 09 06 17" src="https://user-images.githubusercontent.com/11561723/59770069-59630000-9275-11e9-9722-43383e2cadd0.png">

Reply:
👍